tox.ini

```diff
@@ -106,11 +111,11 @@ commands =
     checkformatting: black --check h tests bin
     checkformatting: isort --quiet --check-only h tests bin
     {tests,functests}: sh bin/create-testdb
-    tests: coverage run -m pytest {posargs:tests/h/}
+    tests: python -m pytest --cov --cov-report= --cov-fail-under=0 --numprocesses logical --dist loadgroup {posargs:tests/h/}
     functests: pytest {posargs:tests/functional/}
     docs: sphinx-autobuild -qT --open-browser -b dirhtml -d {envdir}/doctrees docs {envdir}/html
     checkdocs: sphinx-build -qTWn -b dirhtml -d {envdir}/doctrees docs {envdir}/html
-    coverage: -coverage combine
+    coverage: coverage combine
```

Comment:
Removed the `-` here as we have from other projects: we do want tox to fail if `coverage combine` fails